Dockerfile

```diff
@@ -1,133 +1,64 @@
-RUN groupmod -g $NODE_GID node
-
-RUN groupmod -g $PING_GID ping
-
-# Creating local homarr user and group
-RUN groupadd -g $PGID homarr
-RUN useradd homarr -u $PUID -g homarr --home-dir /app --shell /sbin/nologin
-RUN usermod -aG node homarr
-
-# Creating a local Docker group and add docker group to homarr user
-RUN groupadd -g $DOCKER_GID docker
-RUN usermod -aG docker homarr
-
-# Enable sudo for homarr user, only for debug and testing purposes 
-#RUN apk add sudo
-#RUN echo "homarr ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
-
-# Configure entrypoint
-COPY ./docker/entrypoint /
-RUN chmod +x /entrypoint.sh       
-RUN chmod +x /docker-entrypoint.d/*.sh
-
-# Configure supervisord
-COPY ./docker/etc/supervisord.conf /etc/supervisord.conf
-COPY ./docker/etc/supervisor /etc/supervisor
-
-#RUN chown homarr:homarr /app 
-USER node
-WORKDIR /app
-
-COPY --from=compiler --chown=node:homarr /app/next.config.js ./
-COPY --from=compiler --chown=node:homarr /app/public ./public
-COPY --from=compiler --chown=node:homarr /app/package.json ./temp_package.json
-COPY --from=compiler --chown=node:homarr /app/yarn.lock ./temp_yarn.lock
+COPY next.config.js ./
+COPY public ./public
+COPY package.json ./temp_package.json
+COPY yarn.lock ./temp_yarn.lock
 # Automatically leverage output traces to reduce image size
 # https://nextjs.org/docs/advanced-features/output-file-tracing
+COPY .next/standalone ./
+COPY .next/static ./.next/static
+COPY ./scripts/run.sh ./scripts/run.sh
+RUN chmod +x ./scripts/run.sh
+COPY ./drizzle ./drizzle
 
-COPY --from=compiler --chown=node:homarr /app/.next/standalone ./
-COPY --from=compiler --chown=node:homarr /app/.next/static ./.next/static
+COPY ./drizzle/migrate ./migrate
+COPY ./tsconfig.json ./migrate/tsconfig.json
+COPY ./cli ./cli
 
-COPY --from=compiler --chown=node:homarr /app/scripts/run.sh ./scripts/run.sh
-RUN chmod +x ./scripts/run.sh
-COPY --from=compiler --chown=node:homarr /app/drizzle ./drizzle
+RUN mkdir /data
 
-COPY --from=compiler --chown=node:homarr /app/drizzle/migrate ./migrate
-COPY --from=compiler --chown=node:homarr /app/tsconfig.json ./migrate/tsconfig.json
-COPY --from=compiler --chown=node:homarr /app/cli ./cli
+# Install dependencies
+RUN apt update && apt install -y openssl wget
 
 # Move node_modules to temp location to avoid overwriting
 RUN mv node_modules _node_modules
 RUN rm package.json
-
 # Install dependencies for migration
 RUN cp ./migrate/package.json ./package.json
 RUN yarn
-
 # Copy better_sqlite3 build for current platform
 RUN cp /app/node_modules/better-sqlite3/build/Release/better_sqlite3.node /app/_node_modules/better-sqlite3/build/Release/better_sqlite3.node
-
 # Copy node_modules for migration to migrate folder for migration script
 RUN mv node_modules ./migrate/node_modules
+
 # Copy temp node_modules of app to app folder
 RUN mv _node_modules node_modules
 
+RUN echo '#!/bin/bash\nnode /app/cli/cli.js "$@"' > /usr/bin/homarr
+RUN chmod +x /usr/bin/homarr
```

Comment:
I understand that this is meant to be a convenience for `homarr` as a command if it were run in the container, instead of `node /app/cli/cli.js`.

I haven't searched `git blame` or your project history for relevance of what motivated this for the image, but I doubt anyone has been relying on it as you're just writing a fixed single line string here.

The `\n` would only create a new line if you used `echo -e`. So until someone actually wants the convenience of a `homarr` command, perhaps drop this? (_since no one can actually run it, not a breaking change either_)

```suggestion
```